### DISCUSSION TITLE
A complete check of local onedrive data against remote

### DISCUSSION THREAD

**Discussion by addohm** [abraunegg/onedrive#2600] [Q&A] [answered]
Ive had some issues with onedrive on my unraid server recently and I've seen errors pop up during the onedrive sync.  When I restart the onedrive container it doesn't pick up where it left off, or fix any of the errors that I saw.  There _may_ be nothing wrong at all, but I want to be absolutely sure.  That said...

Can I do a complete check against remote without doing an entire sync\resync?  How?

**Comment by abraunegg** [comment, marked as answer]
@addohm 
> Can I do a complete check against remote without doing an entire sync\resync? How?

The *only* way is to do a `--resync` - there is no other way.

**Reply by addohm** [reply]
okie doke.  Thanks :D  Be back in 2-3 weeks!  haha

**Comment by abraunegg** [comment]
@addohm 
If you are running without Docker, I would suggest looking at doing the following:

1. Enable logging to a log file you can review
2. Test 'alpha-5' against your data - it has massive performance increase when doing a resync

Regardless of client version, please note: unless you are downloading all your files again, a resync is just fetching metadata.
